src/App.js

```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
+import React from 'react';
+import Calculator from './components/Calculator';
 import './App.css';
 
-function App() {
+export default function App() {
```

Comment:
- [ ] Please let's make this component class-based to fulfill the requirements of this project and move a step toward approval :walking: